packages/mdc-textfield/index.js

```diff
@@ -203,17 +203,26 @@ class MDCTextField extends MDCComponent {
   }
 
   /**
-   * @return {boolean} True if the Text Field is required.
+   * @return {string} the attribute value from the input element
+   * @param {string} attrName is the name of the attribute on the input element
    */
-  get required() {
-    return this.foundation_.isRequired();
+  getValidationAttribute(attrName) {
```

Comment:
I would still expect this to be exposed via `required` / `pattern` / etc. setters/getters on the component to make it match working with a native input element?

Reply:
are you saying keep `get/set required`, but also add `get/set pattern` that proxy to `foundation_.getValidationAttribute` and `foundation_.setValidationAttribute`? Should we add others eg. minlength, maxlength?

Reply:
Yeah, that's what I was thinking, and yes to min/maxlength too.